### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Fale**
Add EL (CentoOS/Oracle Linux/Scientific Linux.RedHat) e Fedora

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ nudoku is a ncurses based sudoku game.
 #### Arch Linux ####
 I heard rumors of nudoku being in the [AUR](https://aur.archlinux.org/packages/nudoku-git/).
 
+#### EL 7 (CentOS/Oracle Linux/Scientific Linux/RedHat####
+Since nudoku is in the EPEL repository, you yust need to enable the EPEL repository and run:
```
Comment (jubalh) [inline README.md]: there is a small typo: yust -> just :)

File: README.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ nudoku is a ncurses based sudoku game.
...
+
+  # yum install nudoku
+  
+#### Fedora ####
+Since nudoku is in the official Fedora repository, you yust need to run:
```
Comment (jubalh) [inline README.md]: same typo as above

File: README.md [hunk 3]
```diff
@@ -9,6 +9,16 @@ nudoku is a ncurses based sudoku game.
 #### Arch Linux ####
 I heard rumors of nudoku being in the [AUR](https://aur.archlinux.org/packages/nudoku-git/).
 
+#### EL 7 (CentOS/Oracle Linux/Scientific Linux/RedHat####
+Since nudoku is in the EPEL repository, you yust need to enable the EPEL repository and run:
+
+  # yum install nudoku
```
Comment (jubalh) [inline README.md]: This one doesn't get the right formatting could you remove the # and add three times a ` obove and under the line? See in the rest of the README for examples :)

**Comment by jubalh** [comment]
@Fale  Cool! Thank you! I didn't know about that.
Could you fix the small issues I mentioned to your commit and maybe squash your commits, then I'll merge it in :)

**Comment by Fale** [comment]
Thanks for noticing :). I've fixed the errors and now should be ready :). Also, I have already worked on the version 0.2.5 so tonight should go in testing on all Fedora versions and EL7 :)

**Comment by jubalh** [comment]
Thanks!
I squashed your commits into one and merged them in as 1bb2d896c1eff605030b5dbdcd511bb57a8ed38c

Very cool that you brought it to Fedora/RHEL!

**Pull request closed without merging**